.github/workflows/issue_and_pr_commands.yml

```diff
@@ -19,5 +19,5 @@ jobs:
       - name: Run Commands
         uses: ./actions/commands
         with:          
-          token: ${{secrets.GH_BOT_ACCESS_TOKEN}}
+          token: ${{secrets.ISSUE_COMMANDS_TOKEN}}
```

Comment:
Just interested - do these two do the same thing? The only difference I see between them is one excluding dependabot branches 🤔

Reply:
Not sure actually, I just assumed we were using both of them